src/control/lib/control/pool.go

```diff
@@ -842,10 +841,12 @@ func PoolGetProp(ctx context.Context, rpcClient UnaryInvoker, req *PoolGetPropRe
 		pbMap[prop.GetNumber()] = prop
 	}
 
-	for _, prop := range resp {
+	resp := make([]*daos.PoolProperty, 0, len(req.Properties))
+	for _, prop := range req.Properties {
 		pbProp, found := pbMap[prop.Number]
 		if !found {
-			rpcClient.Debugf("DAOS-11418: Unable to find prop %d (%s) in resp", prop.Number, prop.Name)
+			// Properties can be missing due to DAOS-11418 and DAOS-13919
```

Comment:
@knard-intel please can you have a look to see if this fix will disrupt the solution for DAOS-11418 as the change will remove properties that are missing from the response.

Reply:
From my understanding, this should not disrupt the functional tests as we are not using non homogeneous version of DAOS in the test case: creating pool with 22. versions and then using it with a 2.4 versions of dmg.
Moreover, if a end-user is facing such use case, I prefer that the fields which have been renamed do not appear rather than having invalid values.